addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -159,7 +164,7 @@ def _iap_enrich_from_response(self, iap_response):
             lead.message_post_with_view(
                 'iap_mail.enrich_company',
                 values=template_values,
-                subtype_id=self.env.ref('mail.mt_note').id
+                subtype_id=self.env['ir.model.data']._xmlid_to_res_id('mail.mt_note')
```

Comment:
![this-isnt-your-everyday-evil-this-is-advanced-evil](https://user-images.githubusercontent.com/7571158/192256572-cabc0452-6d7a-41d9-a02c-924d8e7009fa.jpg)